internal/pubsub/pubsub.go

```diff
@@ -0,0 +1,62 @@
+// Package pubsub wraps nats calls
+package pubsub
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/nats-io/nats.go"
+
+	"go.infratographer.com/x/pubsubx"
+)
+
+// May be a config option later
+var prefix = "com.infratographer.events"
+
+func newMessage(actorURN string, subjectURN string, additionalSubjectURNs ...string) *pubsubx.Message {
+	return &pubsubx.Message{
+		SubjectURN:            subjectURN,
+		ActorURN:              actorURN, // comes from the jwt eventually
+		Timestamp:             time.Now().UTC(),
+		Source:                "lbapi",
+		AdditionalSubjectURNs: additionalSubjectURNs,
+	}
+}
+
+// PublishCreate publishes a create event
```

Comment:
possibly. there have been lively discussions in slack about what we can do to make interacting with NATS more user friendly. 

One of the ideas we've been kicking around is - for the sake of continuing progress - to use a `internal/x` dir and start to stub out what a good common interface might be in each component. This way we can continue to avoid sticky integration but also aren't gumming up the cogs trying to standardize too early in the infancy of the project.